src/cmap/auth/mongo_credentials.ts

```diff
@@ -37,8 +37,20 @@ export interface AuthMechanismProperties extends Document {
   SERVICE_REALM?: string;
   CANONICALIZE_HOST_NAME?: GSSAPICanonicalizationValue;
   AWS_SESSION_TOKEN?: string;
+  /**
+   * @alpha
```

Comment:
The interesting thing was API extractor was failing when I used @experimental  - it said only @alpha, @beta, @public and @internal were valid.

Reply:
Strange that you'd be seeing this now, since we already have some, api-extractor respects the rules set up in tsdoc.json so we can add it there if you are still seeing errors

Reply:
When I add to tsdoc.json it says the experimental tag already exists. When I just use it then the error is:

```
Error: src/cmap/auth/mongodb_oidc.ts:15:1 - (ae-missing-release-tag) "OIDCMechanismServerStep1" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
Error: src/cmap/auth/mongodb_oidc.ts:25:1 - (ae-missing-release-tag) "OIDCRequestTokenResult" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
Error: src/cmap/auth/mongodb_oidc.ts:32:1 - (ae-missing-release-tag) "OIDCRequestFunction" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
Error: src/cmap/auth/mongodb_oidc.ts:39:1 - (ae-missing-release-tag) "OIDCRefreshFunction" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Reply:
Updated to @public and @experimental